File: android-app/plantpathologyandroid/lib/models/PlantImage.dart
class PlantImage {
  int _id;
  String _plantName;
  String _imageUrl;

  PlantImage(this._plantName, this._imageUrl);

  PlantImage.fromMap(dynamic obj) {
    this._plantName = obj['plantName'];
    this._imageUrl = obj['imageUrl'];
  }

  String get plantName => _plantName;
  String get imageUrl => _imageUrl;

  Map<String, dynamic> toMap() {
    var map = new Map<String, dynamic>();
    map["plantName"] = _plantName;
    map["imageUrl"] = _imageUrl;

    return map;
  }
}

File: android/PlantDiagnosisSystem/app/src/main/java/com/tsi/plantdiagnosissystem/data/model/PlantImage.java
package com.tsi.plantdiagnosissystem.data.model;


import java.io.Serializable;

public class PlantImage implements Serializable {
    int id;
    String plantName;
    String imageUrl;

    public PlantImage(String plantName, String imageUrl) {
        this.plantName = plantName;
        this.imageUrl = imageUrl;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getPlantName() {
        return plantName;
    }

    public void setPlantName(String plantName) {
        this.plantName = plantName;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}
